modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -205,6 +205,41 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					skuOptions="[]"
 				/>
 
+				<c:if test="<%= commerceContext.isRequestQuoteEnabled() %>">
+
+					<%
+					Group group = layout.getGroup();
+
+					String siteDefaultURL = HtmlUtil.escape(group.getDisplayURL(themeDisplay, layout.isPrivateLayout()));
+
+					CommerceCurrency commerceCurrency = commerceContext.getCommerceCurrency();
+					%>
+
+					<clay:button
+						additionalProps='<%=
+							HashMapBuilder.<String, Object>put(
+								"accountId", String.valueOf(commerceAccount.getCommerceAccountId())
+							).put(
+								"channelId", String.valueOf(commerceContext.getCommerceChannelId())
+							).put(
+								"currencyCode", commerceCurrency.getCode()
+							).put(
+								"quantity", 1
+							).put(
+								"siteDefaultURL", siteDefaultURL
+							).put(
+								"skuId", String.valueOf(cpSku.getCPInstanceId())
```

Comment:
For product multi-sku we need to listen the event of the option-selector to verify the specific sku related to the option selected

Reply:
Will be adressed with [COMMERCE-11324](https://issues.liferay.com/browse/COMMERCE-11324), must be removed from this PR